src/components/ListItem/CheckBoxListItem/index.tsx

```diff
@@ -5,21 +5,28 @@ import classNames from 'classnames';
 import styles from '@/components/ListItem/CheckBoxListItem/CheckBoxListItem.css';
 
 const CheckBoxListItem = <T extends {}>({
+  isItemDisabled,
   getIsSelected,
   variablesClassName,
   ...props
 }: Props<T>) => {
   const sizeClass = `list-item-checkbox--${props.size}`;
+
+  const handleCheckedState = (item: any) => {
+    return isItemDisabled ? isItemDisabled : getIsSelected && getIsSelected(item); // eslint-disable-line no-unneeded-ternary
```

Comment:
can we change this to:
`isItemDisabled || (getIsSelected && getIsSelected(item))`